README.md

```diff
@@ -2,13 +2,22 @@
 
 [![GitHub Release](https://img.shields.io/github/v/release/cdr/coder-cli?color=6b9ded&include_prerelease=false)](https://github.com/cdr/coder-cli/releases)
 [![Documentation](https://godoc.org/cdr.dev/coder-cli?status.svg)](https://pkg.go.dev/cdr.dev/coder-cli/coder-sdk)
-![build](https://github.com/cdr/coder-cli/workflows/build/badge.svg)
-[![Go Report Card](https://goreportcard.com/badge/cdr.dev/coder-cli)](https://goreportcard.com/report/cdr.dev/coder-cli)
```

Comment:
What are these deletions?

Reply:
These are build/code badges that are no longer useful since it's close sourced.